src/reader.rs

```diff
@@ -127,7 +127,7 @@ pub fn from_file(path: &Path) -> Result<Config, ConfigError> {
 /// use config::reader::from_str;
 /// use config::error::ConfigErrorKind;
 ///
-/// let parsed = from_str("windows=NO\n");
+/// let parsed = from_str("windows=NO SPACE\n");
```

Comment:
I understand that with this patch `windows=NO\n` is a valid line and as such it breaks the example. But replacing it with `windows=NO SPACE` seems a bit misleading to me, maybe we should find a better example? I think this would make the reader confused, don't you agree?